src/compas/topology/connectivity.py

```diff
@@ -41,6 +46,61 @@ def connectivity_from_edges(edges):
     """"""
     raise NotImplementedError
 
+def join_polylines(polylines, stops = []):
+    """Join polylines. The polylines stop at points connectng more than two lines and to optional additional points.
```

Comment:
typo...

Reply:
from summary not very clear to me what function does. examples?

Reply:
I made it a more atomic and intuitive function: join_lines. The lines are connected into a network from which polylines are extracted as sequences of lines that stop at vertices that are not two-valent. Optional splits can be included. I added an example in the documentation.